satpy/etc/readers/satpy_cf_nc.yaml

```diff
@@ -17,3 +17,4 @@ file_types:
         file_patterns:
          - '{platform_name}-{sensor}-{resolution_type}-{start_time:%Y%m%d%H%M%S}-{end_time:%Y%m%d%H%M%S}.nc'
          - '{platform_name}-{sensor}-{start_time:%Y%m%d%H%M%S}-{end_time:%Y%m%d%H%M%S}.nc'
+         - '{filename}.nc'
```

Comment:
From the little I gathered of the previous conversation the change to the default name in the writer was supported. I'm a little scared how much things will break for users with this change...but yeah.

So with that change included, do we *need* this generic catch-all netcdf pattern?

Reply:
The reason I also added the catch-all pattern was that if the user decides to use his own file naming pattern he won't be able to read them again because the reader does not find any files. So without it the writer/reader loose some flexibility. I personally would be in favour of it since I do like to be able to use my own custom file names. But that is subjective. I guess that is something to be discussed. If it is decided that it should not be included this is definitely something that needs to be documented maybe even with a warning or info box so users are aware of this.

Reply:
Very good point. This seems like a good idea. Thanks.